Makefile

```diff
@@ -35,7 +35,7 @@ run-k6:
 
 .PHONY: build
 build:
-	@CGO_ENABLED=0 go build -ldflags="-w -s" -o ./bin/erpc-server ./cmd/erpc/main.go
+	@CGO_ENABLED=0 go build -ldflags="-w -s" -o ./bin/erpc-server ./cmd/erpc/main.go ./cmd/erpc/validate.go
```

Comment:
Don't think we need this as the main() func only exists in main.go

Reply:
Hmm in that case how about moving it to erpc package? Keeping the main.go only entrypoint